boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -51,6 +51,22 @@
 			};
 		};
 
+		etr_buf_ram0x_region: memory@2f0be000 {
+			compatible = "nordic,owned-memory";
+			reg = <0x2f0be000 DT_SIZE_K(4)>;
+			status = "disabled";
+			perm-read;
+			perm-write;
+			perm-secure;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x2f0be000 0x1000>;
+
+			etr_buffer: memory@0 {
```

Comment:
I think we already discussed it. But I don't remember what is the reason it is not placed under `cpuapp_ram0x_region` that seems to have the same properties.

If the reason is missing features in Secure Domain, please add a TODO comment around to merge these regions in the future.

Reply:
added TODO. Currently this location is also fixed in sdfw. It will be easier to move it around once ETR buffer address is in UICR and not fixed (HM-25088).

Reply:
_etr_buf_ram0x_region_ could still be merged into _cpuapp_ram0x_region_. Preferably at the start then it can fairly easy be disabled without impact when it's not used, otherwise resizing app region wouldn't easily work as this would always sit in the middle. This address here is used for the secure domain as well so if we change it here future sdfw releases will still be aligned.

The uicr change should still be done, sorry for not pushing more on this but I also didn't want to delay this PR here any further which I guess by now would have given enough time.